packages/web/src/alliance/Members.tsx

```diff
@@ -0,0 +1,238 @@
+import { TFunction } from 'next-i18next'
+import * as React from 'react'
+import LazyFade from 'react-lazyload-fadein'
+import { Image, StyleSheet, Text, View } from 'react-native'
+import Ally from 'src/alliance/AllianceMember'
+import { Category as CategoryEnum } from 'src/alliance/CategoryEnum'
+import { H2, H4 } from 'src/fonts/Fonts'
+import { NameSpaces, Trans, useTranslation } from 'src/i18n'
+import { Cell, GridRow, Spans } from 'src/layout/GridRow'
+import { useScreenSize } from 'src/layout/ScreenSize'
+import { ListItem } from 'src/shared/DropDown'
+import DropDownGroup from 'src/shared/DropDownGroup'
+import Outbound, { externalizeURL } from 'src/shared/Outbound'
+import { colors, fonts, standardStyles, textStyles } from 'src/styles'
+
+async function gatherAllies(persistFunc: (data: []) => void, signal: { aborted: boolean }) {
+  const response = await fetch('api/alliance')
+  const alliesByCategory = await response.json()
+  if (!signal.aborted) {
+    persistFunc(alliesByCategory)
+  }
+}
+
+function buildDropDownProps(t: TFunction, currentFilter: string): ListItem[] {
+  return Object.keys(CategoryEnum).map((key) => {
+    return {
+      id: key,
+      selected: key === currentFilter,
+      label: t(`members.categoryTitle.${key.toLocaleLowerCase()}`),
+    }
+  })
+}
+
+function initialState() {
+  return Object.keys(CategoryEnum).map((key: CategoryEnum) => {
+    return {
+      name: key,
+      records: [],
+    }
+  })
+}
+
+const ALL = 'all'
```

Comment:
should this be part of an enum or something like that?

Reply:
i guess it could be part of the CategoryEnum but then thee will be several places it would need to be filtered out which seems extra complicated.